### PR TITLE
added sunken shipwrecks to the ESP

### PR DESCRIPTION
**Pull request by zanzo420**

### CODE REVIEW

File: cDirectX.cpp [hunk 1]
```diff
@@ -533,6 +533,19 @@ int Render()
...
 					ActorArray.push_back(info);
 				}
+				///
+				/// SHIPWRECKS
+				///
+				else if (name.find("BP_Shipwreck_01_a_NetProxy_C") != std::string::npos || name.find("BP_Shipwreck_") != std::string::npos)
+				{
+					info.id = ActorID;
+					info.name = "SHIPWRECK";
+					info.type = animalcrate;
```
Comment (igromanru) [inline cDirectX.cpp:543]: Type animal crate?

**Review by aepkolol — APPROVED** [review]

File: cDirectX.cpp (hunk 1 above)
Comment (zanzo420) [reply cDirectX.cpp:543]: I didn't really know what type to put it under lol, so I just went with animal crate like I did with Sunken Curse statues lol

**Comment by zanzo420** [comment]
Thanks aepkolol!

**Review by EO-Zanzo — APPROVED** [review]
Sunken Shipwrecks show up on the ESP perfectly.

**Pull request merged**